package.json

```diff
@@ -21,18 +21,21 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "start": "electron app --disable-http-cache --no-electron-connect",
+    "start": "node tools/run-dev",
+    "start-app": "electron app --disable-http-cache --no-electron-connect",
```

Comment:
We should remove this? Also, when I run `npm start` for the first time it exits the command (I guess it only compiles) and doesn't start the app.

Reply:
That the bug I fixed last time, can you rebase and try again to check again?
(Also you can do `git clean -xf` to remove compiled files.)

Reply:
@priyank-p we can add this ```git clean -xf``` here as a script. I think that will be good.

Reply:
Sure!

Reply:
Done! For the `git clean` script, I added that as `git clean app/*.js -xf` so it would only remove typescript compiled files and nothing else like `.vscode` etc.